### DISCUSSION TITLE
Max temp set with C104, but printer alarms

### DISCUSSION THREAD

**Discussion by jamiecarl** [mriscoc/Ender3V2S1#1136] [General]
I need to print something at 280C.  Printer can handle it with all-metal hotend which is good for 300+.  I've issued the C104 U1 T315 command and I can set the hotend temp to 280C.  I know the C104 has worked because normally it will fall back to 260C.  As the temperature increases, as soon as it hits 275C the printer throws an alarm saying the "nozzle temperature is too high".

Have I missed a step?  Is this alarm configurable or is this an oversight with setting C104?

**Comment by jamiecarl** [comment]
No one has seen this before or has a solution?

**Comment by mriscoc** [comment]
Check that your heating block has the silicone protection. Also be sure that the hotend fan is not cooling the heating block.
Run MPC (or PID) for the hotend.

**Reply by jamiecarl** [reply]
Thanks for the reply.  Hardware is all good and can handle 280C perfectly fine.  I ran my print at 270C and had no issues, just a bit slower than I know it can handle.  It's just when heating up to 280C, the temperature rises fine but as soon as it hits 275C I get an alarm.

**Reply by jamiecarl** [reply]
Ok, interesting.  I just tried again so I could record a quick video and it didn't do it.   Only change since the original problem was, as you suggested, I ran an MPC autotune.  Lesson learned.

Fixed!  Thanks!
